libr/util/pj.c

```diff
@@ -277,7 +330,6 @@ R_API char *pj_fmt(PrintfCallback p, const char *fmt, ...) {
 	char ch[2] = { 0 };
 	PJ *j = pj_new ();
 	while (*fmt) {
-		j->is_first = true;
```

Comment:
??

Reply:
Yeah not sure how that was removed honestly, wasn't deliberate, re-adding it now - mb.